apps/frontend/src/modules/auth/slices/actions.ts

```diff
@@ -62,4 +66,41 @@ const getAuthenticatedUser = createAsyncThunk<
 	return await authApi.getAuthenticatedUser();
 });
 
-export { getAuthenticatedUser, logOut, signIn, signUp };
+const requestResetPassword = createAsyncThunk<
+	boolean,
+	EmailDto,
+	AsyncThunkConfig
+>(`${sliceName}/send-reset-password-link`, async (emailPayload, { extra }) => {
+	const { authApi, notification } = extra;
+
+	const isSuccessful = await authApi.requestResetPassword(emailPayload);
+
+	if (isSuccessful) {
+		notification.success(NotificationMessage.LINK_SENT);
+	}
```

Comment:
Why don't show notification in auth.slice.ts on fulfilled?

Reply:
Slices can't have any logic and they should only be responsible for changing the state